plugins/services/src/js/components/forms/ArtifactsSection.js

```diff
@@ -71,7 +71,7 @@ class ArtifactsSection extends Component {
               value={item.uri}/>
             <FieldError>{error}</FieldError>
           </FormGroup>
-          <FormGroup className="flex flex-item-align-end column-2 flush-left">
+          <FormGroup isDeleteButton={true} applyLabelOffset={index === 0}>
```

Comment:
I have some concerns for the `isDeleteButton` name in the `FormGroup` since the two concepts are a bit different. I would vote for `narrowMargins` or something like this, that would describe the styling and not the functionality.